app/code/Magento/Store/Test/Unit/Controller/Store/RedirectTest.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Store\Test\Unit\Controller\Store;
+
+use Magento\Store\Api\StoreRepositoryInterface;
+use Magento\Store\Api\StoreResolverInterface;
+use Magento\Store\Model\StoreManagerInterface;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use Magento\Store\Model\StoreResolver;
+
+/**
+ * Test class for \Magento\Store\Controller\Store\SwitchAction
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
+ */
+class RedirectTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * @var \Magento\Store\Controller\Store\SwitchAction
+     */
+    private $model;
+
+    /**
+     * @var StoreRepositoryInterface|\PHPUnit_Framework_MockObject_MockObject
```

Comment:
`PHPUnit_Framework_MockObject_MockObject` should be `PHPUnit\Framework\MockObject\MockObject`